subgraph/origin/package-lock.json

```diff
@@ -184,7 +184,8 @@
         "safe-buffer": {
```

Comment:
Update the package.json with the correct subgraph name.
```
mosaic/{origin|aux}-erc20gateway 
```
You need to run `npm i` again after updating name.

Reply:
Done.